aspnet/mvc/views/partial/sample/src/PartialViewsSample/Controllers/ArticlesController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using PartialViewsSample.ViewModels;
+
+namespace PartialViewsSample.Controllers
+{
+    public class ArticlesController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        public IActionResult Read()
+        {
+            var viewModel = new Article()
+            {
+                AuthorName = "Abraham Lincoln",
+                Title = "The Gettysburg Address",
+                PublicationDate = new DateTime(1863, 11, 19)
+            };
+            viewModel.Sections.Add(new ArticleSection()
+            {
+                Title = "Section One",
```

Comment:
Would it be worth changing this from "Section One" To "Part One"? Probably not - I don't think their is much section ambiguity.